app.rb

```diff
@@ -83,7 +83,7 @@ def create_student
 
     student = Student.new(age, nil, name, parent_permission: permission)
     @people.push(student)
-    puts 'A student created successfully'
+    puts 'A student is created successfully'
```

Comment:
* Kindly fix this error i get when i run the `ruby main.rb` command as nothing happens when i do, and it is stated in the [project requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=Present%20the%20user,quit%20the%20app.) that your `main.rb` file should kindly fix this by extracting all the functionality from the `app.rb` file into a method in the `main.rb` file 

> Present the user with a list of options to perform.
> Lets users choose an option.
> If needed, ask for parameters for the option.
> Have a way to quit the app.
![image](https://user-images.githubusercontent.com/52497246/217883433-dc34aaae-59fe-4b4f-9b06-2fd1ec560558.png)

Reply:
I've updated the main.rb file it should give back the following results as shown below:
![bandicam 2023-02-09 19-32-05-299](https://user-images.githubusercontent.com/90766123/217892336-4447cf5a-e35c-4da4-b6fa-b8b22f63d885.jpg)

I hope everything is in order, if not please don't hesitate to leave a comment or call on zoom.